sunray/Stats.h

```diff
@@ -31,7 +31,6 @@ extern float statMowDistanceTraveled ; // meter
 extern float statTempMin;
 extern float statTempMax;
 
-
```

Comment:
why?

Reply:
because these two vairables are part of the provided MQTT values. BTW: these two lines were last edited by @greymfm